SimTracker/TrackerHitAssociation/src/TrackerHitAssociator.cc

```diff
@@ -99,7 +99,7 @@ void TrackerHitAssociator::makeMaps(const edm::Event& theEvent, const TrackerHit
   //  The collections are specified via ROUList in the configuration, and can
   //  be either crossing frames (e.g., mix/g4SimHitsTrackerHitsTIBLowTof)
   //  or just PSimHits (e.g., g4SimHits/TrackerHitsTIBLowTof)
-
+  const char* highTag = "HighTof";
```

Comment:
It would be even better to make this a ```const char * const```. Then neither the character array nor the pointer would be allowed to change.